packages/volto/src/components/manage/Blocks/Search/components/SearchInput.jsx

```diff
@@ -22,15 +22,15 @@ const SearchInput = (props) => {
     removeSearchQuery,
   } = props;
   const intl = useIntl();
+  const label = data.searchInputPrompt || intl.formatMessage(messages.search);
 
   return (
     <div className="search-input">
       <Input
         id={`${props.id}-searchtext`}
         value={searchText}
-        placeholder={
-          data.searchInputPrompt || intl.formatMessage(messages.search)
-        }
+        aria-label={label}
+        placeholder={label}
```

Comment:
@Gomez there is no need to add the aria-label as well as it will end up duplicating the label of the placeholder and the screen reader will read both.

Although I didn't screen record your change we have a similar situation with the search input of the site.
Have a look at this video where I show how Android 14 talkback reads the input that has
<input type="text" aria-label="Search" name="SearchableText" value="" autocomplete="off" placeholder="Search Site" title="Search">

https://github.com/user-attachments/assets/37f86ff6-85c1-430f-8289-eea9534d0273

Here is how the same input is read by VoiceOver on a mac using Firefox, Edge and Safari

https://github.com/user-attachments/assets/38356ce6-9179-49ec-9408-d8fbd5fe0285

And here is how it sounds without the aria-label

https://github.com/user-attachments/assets/62689156-1841-4a20-a3a7-16a0fcb7e2fb
That sounds better not to have Search Search site which isn't even what shows up in
the Form rotary of VoiceOver

Reply:
> @Gomez there is no need to add the aria-label as well as it will end up duplicating the label of the placeholder and the screen reader will read both.
> 
> Although I didn't screen record your change we have a similar situation with the search input of the site. Have a look at this video where I show how Android 14 talkback reads the input that has
> 
>  talk.back.search.input.mp4 
> Here is how the same input is read by VoiceOver on a mac using Firefox, Edge and Safari
> 
>  input-with-aria-label-with-audio-optimized.mp4 
> And here is how it sounds without the aria-label
> 
> https://github.com/user-attachments/assets/62689156-1841-4a20-a3a7-16a0fcb7e2fb That sounds better not to have Search Search site which isn't even what shows up in the Form rotary of VoiceOver

I've made a demo with the search input from the search block. You can safely use only the placeholder and it will use that information. 

https://github.com/user-attachments/assets/0cd5b57b-3a5b-43bb-be11-b49fa30ef0cc

With your changes aria-label and placeholder has the same value. In theory for Firefox it doesn't repeat the placeholder and aria label if they are the same but it does if they are different.
Haven't tried with Talkback on Android will try some more tonight if I get the time but I remain -1 on the aria-label insertion